mmv1/third_party/terraform/acctest/bootstrap_test_utils.go.tmpl

```diff
@@ -2194,3 +2195,28 @@ func BootstrapSharedTestTagValue(t *testing.T, testId string, tagKey string) str
 
 	return sharedTagValue
 }
+
+type BootstrapClient struct {
+	Name string
+}
+
+func BootstrapIntegrationsGMEKClient(t *testing.T, locationID string) BootstrapClient {
+	config := BootstrapConfig(t)
+	if config == nil {
+		return BootstrapClient{}
+	}
+
+	projectID := envvar.GetTestProjectFromEnv()
+	parent := fmt.Sprintf("projects/%s/locations/%s", projectID, locationID)
+
+	integrationsClient := config.NewIntegrationsClient(config.UserAgent)
+	request := &integrations.GoogleCloudIntegrationsV1alphaProvisionClientRequest{}
```

Comment:
Adding a dependency on an alpha client scares me. Can we do this without using a client at all? The function directly above this has an example of using a direct HTTP call

Reply:
our v1alpha is GA so it should be fine. However I have replaced it wht http call considering the golang client is reflecting older version.